From 43420c9bb90d4290e02bbcaa40c19e00fb347615 Mon Sep 17 00:00:00 2001
From: dp111 <dominic.plunkett@gmail.com>
Date: Sat, 15 Jun 2019 18:19:50 +0100
Subject: [PATCH] bcm2838.dtsi : Correct gic400 memory address ranges

It appears to me the addresses for the gic400 are slightly wrong .  See section 3.2 https://static.docs.arm.com/ddi0471/a/DDI0471A_gic400_r0p0_trm.pdf
---
 arch/arm/boot/dts/bcm2838.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm/boot/dts/bcm2838.dtsi
+++ b/arch/arm/boot/dts/bcm2838.dtsi
@@ -30,8 +30,8 @@
 			compatible = "arm,gic-400";
 			reg =	<0x40041000 0x1000>,
 				<0x40042000 0x2000>,
-				<0x40046000 0x2000>,
-				<0x40048000 0x2000>;
+				<0x40044000 0x2000>,
+				<0x40046000 0x2000>;
 		};
 
 		thermal: thermal@7d5d2200 {
